synapse/config/repository.py

```diff
@@ -22,6 +23,8 @@
 
 from ._base import Config, ConfigError
 
+logger = logging.Logger(__name__)
```

Comment:
I don't know if there's an actual difference, but for consistency with the rest of the codebase:

```suggestion
logger = logging.getLogger(__name__)
```